contracts/staking/src/query.rs

```diff
@@ -55,11 +55,11 @@ pub fn query_config(deps: Deps) -> StdResult<ConfigResponse> {
 
 pub fn query_state(deps: Deps) -> StdResult<StateResponse> {
     let state = STATE.load(deps.storage)?;
-
+    let (redemption_rate, _purchase_rate) = get_rate(&deps);
```

Comment:
```suggestion
    let (redemption_rate, _) = get_rate(&deps);
```

Note: we should `purchase_rate` according to the comment below